docs/architecture-decisions/2024-03-06_local_dev_ui_packaging.md

```diff
@@ -0,0 +1,110 @@
+# Local dev UI packaging
+
+- **Status**: Draft
+- **Owner:** Patrick Dinh (MakerX), Negar Abbasi (MakerX)
+- **Deciders**: Alessandro (Algorand Foundation), Rob Moore (MakerX), MakerX team
+- **Date created**: 2024-03-06
+
+## Context
+
+We are building a web-based local explorer interface to support:
+
+- Transaction / asset / account explorer
+- Dev wallet integration (including KMD support)
+- Network switching
+- ARC-32 app studio
+- Visual transaction representations and basic visual debugging interface to help new users understand what’s happening
+- AlgoKit init and Algokit tasks CLI exploration user interfaces to help new users get the most out of the CLI
```

Comment:
I think we can probably simplif and also make it a bit clearer.

Reply:
I updated this sentence to 
```
- Integration with AlgoKit CLI, for example, calling AlgoKit init and explore AlgoKit tasks
```